### PR TITLE
certificate refresh logic + extend duration

### PR DESCRIPTION
**Pull request by swati-nair**

### CODE REVIEW

File: deploy/crds/operator.ibm.com_v1alpha1_certmanager_cr.yaml [hunk 1]
```diff
@@ -11,5 +11,6 @@ spec:
   enableWebhook: true
   imageRegistry: quay.io/opencloudio
   version: "3.8.2"
+  enableCertRefresh: true
```
Comment (tthorpe2) [inline deploy/crds/operator.ibm.com_v1alpha1_certmanager_cr.yaml:14]: Why did you not include the defaults? 

  enableCertRefresh: true
  refreshCertsBasedOnCA:
  - certName: cs-ca-certificate
    namespace: ibm-common-services
  - certName: mongodb-root-ca-cert
    namespace: ibm-common-services

File: pkg/resources/constants.go [hunk 2]
```diff
@@ -281,6 +282,9 @@ var CSCAIssuerLabelMap = map[string]string{
 //CSCAIssuerName is the name of the CS CA Issuer
 const CSCAIssuerName = "cs-ca-issuer"
 
+//CSCACertName is the name of the CS CA certificate
+const CSCACertName = "cs-ca-certificate"
```
Comment (tthorpe2) [inline pkg/resources/constants.go:286]: How come cs-ca-certificate was added here in eus versus not added in the master stream?

**Review by tthorpe2 — COMMENTED** [review]
2 comments added - one i answered myself

File: pkg/resources/constants.go (hunk 2 above)
Comment (swati-nair) [reply pkg/resources/constants.go:286]: it exists in the master branch: https://github.com/IBM/ibm-cert-manager-operator/blob/master/pkg/resources/constants.go#L286

Comment (tthorpe2) [reply pkg/resources/constants.go:286]: good. must have pulled up the wrong version

**Review by tthorpe2 — APPROVED** [review]
/lgtm

**Pull request merged**